manifests/e2etests.yaml

```diff
@@ -367,8 +367,15 @@ spec:
         env:
         - name: SECONDS_BETWEEN_RUNS
           value: "300"
+      - name: dns
+        <<: *e2etests-image
+        command: ["python"]
+        args: ["kubee2etests/scripts/test_runner.py", "dns"]
+        env:
+        - name: SECONDS_BETWEEN_RUNS
+          value: "300"
         - name: TEST_NAMESPACE
-          value: kube-e2etests-deployment-scale-service
```

Comment:
It looks like you've taken out the `TEST_NAMESPACE` variable from the `deployment-scale-service` - did you mean to do that?